.github/workflows/release.yaml

```diff
@@ -40,3 +40,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - run: gh workflow run ci.yml --repo nextstrain/docker-base
+  create-github-release:
+    needs: [run]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.inputs.version}}
+    - name: Create GitHub Release
+      run: |
+        gh release create \
+          "${{ github.event.inputs.version}}" \
+          --repo nextstrain/augur \
+          --title "${{ github.event.inputs.version}}" \
+          --target "$(git rev-list -1 ${{ github.event.inputs.version}})" \
+          --notes "See the [official changelog](https://github.com/nextstrain/augur/blob/master/CHANGES.md) for details on what's new in this version."
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH }}
```

Comment:
_non-blocking_

Instead of using this separate token, I wonder about instead elevating the [permissions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions) of the default/automatically provisioned token for the workflow? For example, I believe setting:

```yaml
permissions:
  metadata: read
  contents: write
```

at the job-level would be sufficient, as [creating a release is part of the `contents:write` scope](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-contents). (The `metadata: read` is implicit and not needed according to docs, but clearer to be explicit here I think.)

Upside of this is less secrets/tokens to manage, but a (minor?) downside is probably that the release gets created as some GitHub Actions user instead of our Nextstrain Bot.

Reply:
Scratch that; the permissions thing is unnecessary as our repos by default use the permissive defaults instead of the restrictive defaults, so the normal `GITHUB_TOKEN` already has permissions to create releases. The only thing to do is map it into the environment for that step:

```yaml
env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

The caveat about creating the release as github-actions vs. nextstrain-bot still applies, of course.

Reply:
Updated to use the normal `GITHUB_TOKEN` as done in https://github.com/nextstrain/cli/pull/186.